proto/internalapi/central/collector_runtime_config.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package central;
+
+import "storage/collector_runtime_config.proto";
+
+option go_package = "./internalapi/central;central";
+
+message CollectorRuntimeConfigACK {
+  storage.CollectorRuntimeConfigACK collector_runtime_config_ack = 1;
```

Comment:
Even though this is an internal API, it is a bad pattern to reference storage protos in an API. This leaks our storage into the API layer and limits our ability to change how things are stored without impacting the API.  I recommend creating an API version of these objects which will require a convert function when it is time to use them.  Same comment for the message below.

Reply:
I moved the new protobuf definitions I have from `storage` to `internalapi`, but that doesn't solve the problem as I am using `ResourceCollection` which is defined in storage. Does `ResourceCollection` need to be duplicated and copied into `internalapi` as well?

Reply:
Yes and the user.  This proto should not import from `storage`.

Reply:
Nothing is imported from `storage` anymore.